temporalio/workflow.py

```diff
@@ -424,6 +425,17 @@ class ParentInfo:
     workflow_id: str
 
 
+@dataclass(frozen=True)
+class UpdateInfo:
+    """Information about a workflow update."""
+
+    id: str
+    """Update ID."""
+
+    name: str
+    """Update type name."""
```

Comment:
This meaning is unclear to me.  Is name the function name (unscoped by the class name) unless it's overridden with the attribute?

Reply:
Yes. The meaning of update ID above it may be unclear to users too.

The "update name" is a Temporal concept like "workflow type" in "Info.workflow_type". Ideally the general documentation would detail this, but if necessary we can update all of the Python docs for signal/update/query name, all the workflow info stuff, and all of that if we want. But this one field isn't unique (it applies to the field above it, where "name" is used elsewhere, all the other fields we delegate to docs to describe, etc).